### PR TITLE
[WIP] Migrate Dashboard page

### PR DESCRIPTION
**Pull request by mickaelandrieu**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | POC of Dashboard page migration.
| Type?         | new feature
| Category?     | BO
| BC breaks?    | yes (dashboard modules templates need to use boostrap4 and not boostrap3)
| Deprecations? | yes (bootstrap 3 templates).
| How to test?  | Access admin-dev/dash/ url

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/9406)

### CODE REVIEW

File: src/PrestaShopBundle/Resources/views/Admin/dashboard.html.twig [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+ * International Registered Trademark & Property of PrestaShop SA
+ *#}
+{% extends '@PrestaShop/Admin/layout.html.twig' %}
+{% trans_default_domain "Admin.Dashboard.Feature" %}
+
+{% block content %}
+    <script>
+        var dashboard_ajax_url = {{ path('admin_dashboard') }};
+        var adminstats_ajax_url = {{ getAdminLink('AdminStats') }};
+        var no_results_translation = {{ "No result"|trans }};
```
Comment (LouiseBonnard) [inline src/PrestaShopBundle/Resources/views/Admin/dashboard.html.twig:32]: It is currently in the _messages.xfl_ file. Perhaps _Admin.Global_?

File: src/PrestaShopBundle/Resources/views/Admin/dashboard.html.twig [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+        var dashboard_use_push = 5;
+        var read_more = {{ "Read more"|trans }};
```
Comment (LouiseBonnard) [inline src/PrestaShopBundle/Resources/views/Admin/dashboard.html.twig:34]: It is located in _Admin.Actions_.

**Comment by matks** [comment]
@mickaelandrieu will you finish this PR or do you want someone else to finish it as you are focused on other topics now ?

**Comment by matks** [comment]
> @mickaelandrieu will you finish this PR or do you want someone else to finish it as you are focused on other topics now ?

🆙 @mickaelandrieu 😉

**Comment by mickaelandrieu** [comment]
I don't know, depends mainly on my future missions and on what the product team want to do with the Dashboard page /c @colinegin

**Comment by colinegin** [comment]
Yes we need it to be migrated for 1.7.6, as we are going to work on dashboard modules in the meantime.
Let me know if you want to discuss about it.

**Comment by mickaelandrieu** [comment]
Hello @matks, I don't think I'll have time to migrate this page during 1.7.6.x development period so I'm gonna close this one *but* I don't delete the branch.

Feel free to reuse any content here :+1:

**Comment by matks** [comment]
Thanks for the hard work @mickaelandrieu 😃

I will finish what you started !

(guess the movie quote 😅 ?)

**Comment by RubenMartins** [comment]
Great work with this PR, just wondering when the dashboard will get this much needed facelift 💅 
Can we expect something for 1.7.8? or more likely down the road say 1.7.9 or 1.7.10 maybe?

**Comment by matks** [comment]
Hi @RubenMartins unfortunately migrating (and the facelifting) of this page has been postponed as we found out that, the day we migrate this page we also need to migrate (and facelift) all of the modules being displayed on the Dashboard. So it's a massive work 😅 .

It will not be done for 1.7.8 . I don't think 1.7.9 either 🤔 . It's scheduled for sometime in the future but I'm not sure it's milestoned for a version yet.

**Comment by RubenMartins** [comment]
Thank you for the reply @matks. 
Is there a list of modules to convert before we get to the dashboard? to see if I can help you guys with that.

**Comment by matks** [comment]
@RubenMartins Well, it's all the modules being displayed on the dashboard. Maybe @MatShir has the complete list.



Each module needs 2 updates:
1. When page is migrated to Symfony, some extension mechanisms (like hooks) might change and module needs to be updated to continue working as expected with new version
2. When page is migrated to Symfony, the theme is changed. Some JS is updated, Bootstrap version is higher, the layout changes. So the module needs to be updated to "fit in" the new design, so that it looks decent. It's just about keeping the global design homogenous.

**Pull request closed without merging**